tests/tests.go

```diff
@@ -1295,8 +1296,21 @@ func (ctl *controller) listBackups(databaseID int) ([]*api.Backup, error) {
 	return backups, nil
 }
 
+func (ctl *controller) getBackup(databaseID, backupID int) (*api.Backup, error) {
+	backupList, err := ctl.listBackups(databaseID)
+	if err != nil {
+		return nil, err
+	}
+	for _, backup := range backupList {
+		if backup.ID == backupID {
+			return backup, nil
+		}
+	}
+	return nil, errors.Errorf("Failed to find backup with ID %d", backupID)
+}
+
 // waitBackup waits for a backup to be done.
-func (ctl *controller) waitBackup(databaseID, backupID int) error {
+func (ctl *controller) waitBackup(databaseID, backupID int, cond func(*api.Backup) bool) error {
```

Comment:
What does `cond` mean?

Reply:
condition. Changed to `condition`.

Reply:
Yes. It says under what condition we stop waiting for the backup.

Reply:
I think `shouldWait` may be a better name.

Reply:
SG.